Ginger/Ginger/Actions/ActScriptEditPage.xaml.cs

```diff
@@ -120,10 +121,67 @@ private void ScriptInterpreterComboBox_SelectionChanged(object sender, Selection
             if (ScriptInterpreterComboBox.SelectedValue.ToString() == "Other")
             {
                 InterpreterPanel.Visibility = Visibility.Visible;
+                InterpreterPanelLabel.Visibility = Visibility.Visible;
+                ScriptNameComboBox.Items.Clear();
+                if (!Directory.Exists(SHFilesPath))
+                    Directory.CreateDirectory(SHFilesPath);
+                string[] fileEntries = Directory.EnumerateFiles(SHFilesPath, "*.*", SearchOption.AllDirectories)
```

Comment:
Let's create a function with Folder and file extension as an input and return all files with this extension.